### PR TITLE
add announcement

### PR DESCRIPTION
**Pull request by kostyapenkov**

### CODE REVIEW

File: maintenance/2023-10-29.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## API maintenance
+
+On Sun, 2023-10-29, between 05:00 and 10:00 UTC Database service for [Foundries.io API](https://api.foundries.io/ota/) will be reconfigured. During the reconfiguration, some calls to the API might fail.
```
Comment (mike-scott) [inline maintenance/2023-10-29.md:3]: ```On Sun, 2023-10-29, between 05:00 and 10:00 UTC: The database service for [Foundries.io API](https://api.foundries.io/ota/) will be reconfigured. ```

File: maintenance/2023-10-29.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Affected calls:
+ - device list
+ - waves list
+ - target tests
+ - device tests
+
+As a consequence, web application [app.foundries.io](https://app.foundries.io/) might diplay errors.
```
Comment (mike-scott) [inline maintenance/2023-10-29.md:11]: As a consequence, our web application [app.foundries.io](https://app.foundries.io/) might display errors.

File: maintenance/2023-10-29.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Expected downtime - less than 5 minutes in a row.
```
Comment (mike-scott) [inline maintenance/2023-10-29.md:13]: "in a row" seems complicated.  Does this mean that they might see several bits of downtime each not to exceed 5 minutes?

Comment (Tim-Anderson) [reply maintenance/2023-10-29.md:13]: If that is the case then it should be "not to exceed 5 minutes at a time"

Comment (kostyapenkov) [reply maintenance/2023-10-29.md:13]: > "in a row" seems complicated. Does this mean that they might see several bits of downtime each not to exceed 5 minutes?

its correct

**Pull request merged**